### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by cojoj**
I've added a mention about sample project I've just created which you can find [here](https://github.com/cojoj/XCSDemo).

When I was working on `XCSDemo` I've noticed there are some missing points in `README`, so this PR also updates those parts of `README` - now everything should be accurate 👌

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,8 @@ First brought to you in [Buildasaur](https://github.com/czechboy0/Buildasaur), n
 This is an unofficial, community-maintained project and is not associated with Apple. We also maintain the **[unofficial documentation](https://github.com/czechboy0/XcodeServer-API-Docs) of Xcode Server** in case you'd like to integrate using a different programming language! :wink:
 
 # :mortar_board: Getting Started With Xcode Server 
-To find out how to set up Xcode Server on your Mac in minutes (and more), check out my [series of tutorials](http://honzadvorsky.com/pages/xcode_server_tutorials/).
+To find out how to set up Xcode Server on your Mac in minutes (and more), check out my [series of tutorials](http://honzadvorsky.com/pages/xcode_server_tutorials/).  
+If you want to check how easy it is to integrate `XcodeServerSDK` with iOS app, check this little [sample project](https://github.com/cojoj/XCSDemo).
```
Comment (czechboy0) [inline README.md]: How about: "To see how easy it is to integrate `XcodeServerSDK` into an iOS app, check out the [XCSDemo](https://github.com/cojoj/XCSDemo) project created by [@cojoj](https://github.com/cojoj)."

File: README.md [hunk 2]
```diff
@@ -37,14 +38,22 @@ pod 'XcodeServerSDK'
 
 Create the server config object with the server's URL, username and password.
 ```swift
-let config = XcodeServerConfig(host: "https://127.0.0.1", user: "IRuleBots", password: "superSecr3t")
-let server = XcodeServerFactory.server(config)
+do {
+    let config = XcodeServerConfig(host: "https://127.0.0.1", user: "IRuleBots", password: "superSecr3t")
+} catch {
+    fatalError("Something went wrong!")
```
Comment (czechboy0) [inline README.md]: We don't want to encourage ignoring errors here, how about

```
fatalError("Couldn't create config, error \(error)")
```

**Comment by buildasaur** [comment]
## Result of Integration **1**

_Duration_: 1 minute and 13 seconds
_Result_: **Perfect build!** All 73 tests passed. :+1:
_Test Coverage_: 56%.

File: README.md [hunk 3]
```diff
@@ -37,14 +38,22 @@ pod 'XcodeServerSDK'
...
+}
+```
+
+Instantiate `XcodeServer`.
+```swift
+let server = XcodeServer(config: config, endpoints: endpoints)
```
Comment (czechboy0) [inline README.md]: There's a convenience method: `let server = XcodeServerFactory.server(config: config)`

**Comment by czechboy0** [comment]
Thanks @cojoj, looks good, I just added a couple of comments.

File: README.md (hunk 2 above)
Comment (cojoj) [inline README.md]: We can also write something more detailed to show what are the possibilities, like:

``` swift
do {
    let config = XcodeServerConfig(host: "https://127.0.0.1", user: "IRuleBots", password: "superSecr3t")
} catch ConfigurationErrors.NoHostProvided {
    fatalError("You haven't provided any host")
} catch ConfigurationErrors.InvalidHostProvided(let host){
    fatalError("You've provided invalid host: \(host)")
} catch ConfigurationErrors.InvalidSchemeProvided(let scheme) {
    fatalError("You've provided invalid scheme: \(scheme)")
} catch {
    fatalError("Error, not related to XcodeServerConfig; \(error)")
}
```

File: README.md (hunk 3 above)
Comment (cojoj) [inline README.md]: Where? I can't find it here: https://github.com/czechboy0/XcodeServerSDK/blob/master/XcodeServerSDK/XcodeServer.swift

Comment (czechboy0) [inline README.md]: https://github.com/czechboy0/XcodeServerSDK/blob/master/XcodeServerSDK/XcodeServerFactory.swift#L13

File: README.md (hunk 2 above)
Comment (czechboy0) [inline README.md]: Yeah that looks nicer!

File: README.md (hunk 3 above)
Comment (cojoj) [inline README.md]: Totally forgot about `XcodeServerFactory`... 😬

**Comment by czechboy0** [comment]
Great, thanks @cojoj!

**Pull request merged**